Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM alpine:latest as builder
 MAINTAINER b3vis
 ARG BORG_VERSION=1.1.10
-ARG BORGMATIC_VERSION=1.3.14
+ARG BORGMATIC_VERSION=1.4.0
```

Comment:
will create a new release and image tag for this

Reply:
Yup, `1.4.0` is the release that adds the PostgeSQL support.